### ISSUE TITLE
Total val loss as training problem indicator

### ISSUE THREAD

**Issue by yogaaltariz** [ultralytics/yolov5#4474]
in results.txt there is the total loss value of the three training losses. My question is, can we use the total train loss and also total of the validation losses (by adding manually) to diagnose overfitting or underfitting problem? 

I have tried to plot the total train loss and total val loss like this
![image](https://user-images.githubusercontent.com/43367856/129932220-9cc9ccb0-b5cf-456d-bb87-f382d938996f.png)

as we can see at the curve above, there is a gap between train loss and val loss

**Comment by glenn-jocher** [comment]
@yogaaltariz results.txt has been replaced with results.csv using our new CSV Logger which no longer tracks total loss. You can create total loss yourself as the sum of the loss components. Overfitting occurs independently at different times in each loss component.

<img width="1909" alt="Screenshot 2021-08-18 at 18 16 13" src="https://user-images.githubusercontent.com/26833433/129934244-04b48851-5e82-45f9-99b9-089266ef4f6c.png">

**Comment by yogaaltariz** [comment]
Ok got it, thanks @glenn-jocher
